.travis.yml

```diff
@@ -1,7 +1,26 @@
+# Created with https://github.com/dart-lang/mono_repo
 language: dart
-sudo: false
+
 dart:
-  - stable
   - dev
-script:
-  - cd app && pub get && pub run test
+  - stable
+
+env:
+  - PKG=app TASK=dartanalyzer
```

Comment:
Normally I'd rather call this "SUBFOLDER" or something like this, because it's only use is for pushd, though it is generated ...

Reply:
there tends to be a one-to-en w/ the dir name and the pkg name. I hear ya.